app/Filament/Resources/InterpretationResource/Pages/EditInterpretation.php

```diff
@@ -16,4 +17,31 @@ protected function getActions(): array
             Actions\DeleteAction::make(),
         ];
     }
+
+    protected function getFormSchema(): array
+    {
+        return [
+            Forms\Components\TextInput::make('name')
+                ->required()
+                ->maxLength(255)
+                ->columnSpan(2),
+            Forms\Components\TextInput::make('route')
+                ->required()
+                ->maxLength(255),
+            Forms\Components\TextInput::make('namespace')
+                ->maxLength(255),
+            Forms\Components\Toggle::make('route_has_params')
```

Comment:
Could we potentially comment out this part and put it back later when we need the option?

Reply:
I left it there but disabled it so that we could present the state in the UI without making it interactive, but I guess we could remove it as well.